src/Components/Search/Previews/Grids/ArtistSearch/NoResults.tsx

```diff
@@ -0,0 +1,42 @@
+import { Flex, Serif } from "@artsy/palette"
+import React from "react"
+
+const ArtworkIcon: React.SFC = () => {
+  return (
+    <svg width="18px" height="18px" viewBox="2 2 14 14" version="1.1">
+      <title>Artwork</title>
+      <desc>Artwork icon</desc>
+      <g
+        id="Indicator/Artwork/Md"
+        stroke="none"
+        strokeWidth="1"
+        fill="none"
+        fillRule="evenodd"
+      >
+        <path
+          d="M5.68794719,6 L9.04687494,2.24412276 L12.3137773,6 L15,6 L15,15 L3,15 L3,6 L5.68794719,6 Z M7.02951148,6 L10.9884205,6 L9.03645839,3.75587724 L7.02951148,6 Z M4,7 L4,14 L14,14 L14,7 L4,7 Z M6,9 L6,12 L12,12 L12,9 L6,9 Z M5,8 L13,8 L13,13 L5,13 L5,8 Z"
+          id="Combined-Shape"
+          fill="#000000"
+          fillRule="nonzero"
+        />
+      </g>
+    </svg>
+  )
+}
+
+export const NoResultsPreview: React.SFC = () => {
+  return (
+    <Flex
+      justifyContent="center"
+      alignItems="center"
+      flexDirection="column"
+      my={140}
+      pr={3}
```

Comment:
We are setting `pr={3}` within this component because `pl={3}` is set by a parent component. I'm going to follow with a refactor of the styles such that child components don't need to know how much spacing to put on one side, and it's all handled by the container.